.idea/libraries/jetty.xml

```diff
@@ -1,7 +1,7 @@
 <component name="libraryTable">
   <library name="jetty">
     <CLASSES>
-      <root url="jar://$PROJECT_DIR$/build/third_party/java/jetty/jetty-repacked.jar!/" />
+      <root url="jar://$PROJECT_DIR$/third_party/java/jetty/jetty-repacked.jar!/" />
```

Comment:
I see, but I still think that the path should point to `third_party/` and not to `build/third_party/`

```
$ git clean -xdf && ./go clean && find . -name jetty-repacked.jar
-- no result whatsoever
$ ./go clean test_firefox && find . -name jetty-repacked.jar
-- still no result
$ ./go repack-jetty
$ find . -name jetty-repacked.jar
./build/third_party/java/jetty/jetty-repacked.jar
./third_party/java/jetty/jetty-repacked.jar
-- there it is!
$ ./go clean
$ find . -name jetty-repacked.jar
./third_party/java/jetty/jetty-repacked.jar
-- gone again!
```

Looks like you get `build/third_party/java/jetty/jetty-repacked.jar` right after executing `go repack-jetty`; but as soon as you run `go clean` it is lost, and all is left is `third_party/java/jetty/jetty-repacked.jar`.